src/test/kotlin/no/nav/helse/GCS.kt

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
What is this file?

Reply:
Denne filen er ment å emulere en GCS server, men den støttet dessverre ikke oppretting av lifecycle på bucket, og ga 404 når GcpStorageBucket ble instansiert. Der er der vi verfiserer at lifecycle policy er satt på bucket. Inntil videre har jeg lagt kommenrert den ut. Var ment å brukes til testing.